CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Breaking Changes
+
+- Bumped minimum macOS version from 10.14.0 to 10.15.0
```

Comment:
`h`: We can not do this due to https://github.com/getsentry/sentry-cocoa/issues/6758